### ISSUE TITLE
AU and NZ

### ISSUE THREAD

**Issue by s-digweed** [dtankdempse/free-iptv-channels#18]
Hi Reuben,

Can you extend your amazing work to include Matt's Australia and NZ playlists (separate). These doing a 404 now.

No rush, whenever you have time.

Thanks for sublime work.

**Comment by dtankdempse** [comment]
Yes, I'll take a look at some of his other Kodi addons now that I have them. :)

**Comment by dtankdempse** [comment]
You want to test these out and let me know before I add them to the other scripts?

`https://tinyurl.com/multiservice21?service=NZ`

`https://tinyurl.com/multiservice21?service=AU`

Thanks.

**Comment by joaquinito2070** [comment]
It's working.

**Comment by s-digweed** [comment]
Brilliant, thanks man!
